roles/ntc/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
I'd rather not have a role that is just a single task which is calling the only module here.  Want to ensure this isn't over-engineered (although maybe it does get added in the future)...but now it's just being used in the verify pb and for testing anyway.  See other notes for more clarity.